lib/alchemy/test_support/essence_shared_examples.rb

```diff
@@ -7,15 +7,12 @@
 
   it "touches the content after update" do
     element = create(:alchemy_element)
-    content = create(:alchemy_content, element: element)
+    content = create(:alchemy_content, element: element, updated_at: 3.days.ago)
     essence.save
     content.update(essence: essence, essence_type: essence.class.name)
-    date = content.updated_at
-    Timecop.travel(5.minutes.from_now) do
-      content.essence.update(essence.ingredient_column.to_sym => ingredient_value)
-      content.reload
-      expect(content.updated_at).not_to eq(date)
-    end
+    content.essence.update(essence.ingredient_column.to_sym => ingredient_value)
```

Comment:
Absolutely right, I think the whole test was a bit murky, so I've re-arranged it, and have verified that it fails if I comment out this line:

- https://github.com/AlchemyCMS/alchemy_cms/blob/master/lib/alchemy/essence.rb#L201

Thanks for looking!